src/libraries/grommet/components.js

```diff
@@ -2176,6 +2237,7 @@ export const components = {
   DataTableColumns: {
     component: DataTableColumns,
     name: 'DataTableColumns',
+    documentation: 'https://v2.grommet.io/datatablecolumns',
     defaultProps: {
       drop: true,
```

Comment:
```suggestion
```
The default behavior is `drop={false}`

can you also update on line 2246 to have `drop: false` instead?

Reply:
@Sulaymon333 I don't see this comment incorporated, this line should be removed I think.